src/main/java/eu/deltasource/internship/hotelmanagementsystem/NoRoomsAvailableException.java

```diff
@@ -0,0 +1,8 @@
+package eu.deltasource.internship.hotelmanagementsystem;
+
+public class NoRoomsAvailableException extends  RuntimeException{
```

Comment:
Code is not formatted, also can you add some trivial javadoc, like "Exception when no available rooms are found in the Hotel"